site/src/util/formUtils.ts

```diff
@@ -49,7 +51,7 @@ export const getFormHelpers =
 export const getFormHelpersWithError = <T>(
   form: FormikContextType<T>,
   error?: Error | unknown,
-): ((name: keyof T, HelperText?: ReactNode) => FormHelpers) => {
+): ((name: keyof T, HelperText?: ReactNode, errorName?: string) => FormHelpers) => {
```

Comment:
nit: Instead of ‘errorName’, we can call the argument field ‘fieldName’ or ‘backendFieldName’.